### PR TITLE
VaNotify - Add confirmation email to 5490

### PR DESCRIPTION
**Pull request by nathanbwright**
## Summary

- Adds confirmation email to form 5490
- Sending email is behind a feature flag which will be removed once once we have incrementally rolled this out and verified that it is working as expected.

## Related issue(s)
- https://app.zenhub.com/workspaces/forms-strike-team-620c2914e703390013c4b414/issues/gh/department-of-veterans-affairs/vanotify-team/692

## Testing done

- Added automated testing for sending confirmation email to the `SavedClaim::EducationBenefits`.
- We will do testing in staging before rolling this out in prod.

## Screenshots
_Note: Optional_

## What areas of the site does it impact?
Kicks off an sidekiq job to send an email confirmation to veterans when they have successfully submitted a 5490 form.

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Comment by rjohnson2011** [comment]
Just wanted to check in on the status of this since the PR has been open for a while and we're trying to clean up the list of open PRs. Thanks!

**Comment by nathanbwright** [comment]
> Just wanted to check in on the status of this since the PR has been open for a while and we're trying to clean up the list of open PRs. Thanks!

Just waiting on a review from someone in platform support. https://dsva.slack.com/archives/CBU0KDSB1/p1674170931544739

**Review by rjohnson2011 — APPROVED** [review]

**Review by markkvba — APPROVED** [review]

**Pull request merged**
